.changeset/beige-trainers-run.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/backend-common': minor
+'@backstage/integration': minor
```

Comment:
These should be patch changes. Since we are still in 0.x releases, minor is only for breaking changes, and patch for new features!

Reply:
Changed this to a patch!